WtWSMS/history/characters/vandalic.txt

```diff
@@ -294,7 +294,8 @@
 	dynasty = 1042171
 	religion = "arian"
 	culture = "vandalic"
-	356.1.1 = {
```

Comment:
FMG claims (https://fmg.ac/Projects/MedLands/VANDALS,%20SUEVI,%20VISIGOTHS.htm) that Godigisel was a son of one Radagaisus and Cella, although it does not give a primary source for this statement. But IMHO Radagaisus could be added as intermediate generation between Visimar and Godigisel because of some age inconsistencies, but of course it's up to you. I personally don't believe that Godigisel was that old at the time of his death.

Reply:
right, I'll add Radagaisus as intermediate generation to have a more congruent age for Godegisel